provision/service.go

```diff
@@ -67,7 +75,7 @@ func New(cfg Config, sdk SDK.SDK, logger logger.Logger) Service {
 }
 
 // Provision is provision method for adding devices to proxy.
-func (ps *provisionService) Provision(token, externalID, externalKey string) (res Result, err error) {
+func (ps *provisionService) Provision(token, extID, externalKey string) (res Result, err error) {
```

Comment:
`externalID` was fine

Reply:
it overlaps with constant `externalID`

Reply:
Change the constant to `externalIDKey` or something similar